code/__DEFINES/stat_tracking.dm

```diff
@@ -44,6 +44,10 @@
 
 #define SET_COST_LINE(...) SET_COST("[__LINE__]")
```

Comment:
should redefine set_cost_line to be as the second half of the below, and define set_cost_stmt in terms of it

Reply:
As in SET_COST_LINE should also include the code in its text? Not sure that is a good idea since it only uses `(...)` so you can `SET_COST_LINE()`, and if you want to put anything inside then `SET_COST_STMT` is waht you want anyway

Reply:
ah right shit I forgot. nvm